src/lib.rs

```diff
@@ -119,16 +119,28 @@ impl HoverData {
             .map(|(idx, _)| idx)
             .unwrap();
 
-        let (target, other) = match position {
-            0 => (None, Rect::EVERYTHING),
-            1 => (Some(Split::Left), Rect::everything_left_of(center.x)),
-            2 => (Some(Split::Right), Rect::everything_right_of(center.x)),
-            3 => (Some(Split::Above), Rect::everything_above(center.y)),
-            4 => (Some(Split::Below), Rect::everything_below(center.y)),
+        let (tab_dst, other) = match position {
+            0 => (TabDestination::Append, Rect::EVERYTHING),
```

Comment:
As a further improvement, we could replace these magic numbers with constants.

Reply:
I removed the magic number indexing, tell me what you think!

Reply:
Looks good!